build-cluster-ec2.yml

```diff
@@ -106,14 +207,37 @@
   tasks:
     - include: python/tasks/main.yml
     - include: java/tasks/main.yml
+    - include: ntp/tasks/main.yml
     - include: docker/tasks/main.yml
       when: "{{ INSTALL_DOCKER }} == true"
+    - include: conductr/tasks/install-agent.yml
```

Comment:
If we have private nodes declared as `CONDUCTR_PRIVATE_AGENT_ROLES`, we can then pass the roles into the agent task as such:

```
- include: conductr/tasks/install-agent.yml
   vars:
     CONDUCTR_AGENT_ROLES={{ CONDUCTR_PRIVATE_AGENT_ROLES }}
```